Luminary099/POWERED_FLIGHT_SUBROUTINES.agc

```diff
@@ -1,6 +1,6 @@
 # Copyright:	Public domain.
 # Filename:	POWERED_FLIGHT_SUBROUTINES.agc
-# Purpose: 	Part of the source code for Luminary 1A build 099.
+# Purpose:	Part of the source code for Luminary 1A build 099.
```

Comment:
This appears to be a tab, only spaces are used in the code comments, only the uncommented code should use tabs.